src/anm/media/audio.js

```diff
@@ -349,5 +329,48 @@ Audio.prototype.connect = function(element) {
     element.on(C.S_STOP, stop);
     element.on(C.S_PAUSE, stop);
 };
+/**
+ * @method clone
+ *
+ * @return {anm.Audio}
+ */
+Audio.prototype.clone = function() {
+    var clone = new Audio('');
+    clone.url = this.url;
+    clone.offset = this.offset;
+    return clone;
+};
+
+// workaround, see http://stackoverflow.com/questions/10365335/decodeaudiodata-returning-a-null-error
+function syncStream(node){
```

Comment:
I just moved @alexeypegov's code from top to bottom in this case :). I just prefer to place class code above the util functions, it was the reason.